mlflow/store/tracking/dbmodels/models.py

```diff
@@ -598,7 +598,7 @@ class SqlInputTag(Base):
     __tablename__ = "input_tags"
     __table_args__ = (PrimaryKeyConstraint("input_uuid", "name", name="input_tags_pk"),)
 
-    input_uuid = Column(String(36), ForeignKey("experiments.experiment_id"), nullable=False)
+    input_uuid = Column(String(36), ForeignKey("inputs.input_uuid"), nullable=False)
```

Comment:
also a typo from earlier

Reply:
Any migration typos, or just in the model?